src/core/compact_object_test.cc

```diff
@@ -731,4 +732,69 @@ static void BM_UnpackSimd(benchmark::State& state) {
 }
 BENCHMARK(BM_UnpackSimd);
 
+static void BM_LpCompare(benchmark::State& state) {
+  std::random_device rd;
+  uint8_t* lp = lpNew(0);
+  for (unsigned i = 0; i < 100; ++i) {
+    lp = lpAppendInteger(lp, rd() % (1ULL << 48));
+  }
```

Comment:
I was about to comment that is used below but IMO it's just a test and only a few lines so who cares